docker/docker-compose.build-pdf.yaml

```diff
@@ -0,0 +1,13 @@
+version: '3.8'
+
+services:
+  pdf-builder:
+    build:
+      context: ../
+      dockerfile: docker/images/Dockerfile.pdf-builder
+    image: edb/x-docs-md-to-pdf
```

Comment:
@josh-heyer @cero-miedo Is there an established EDB Docker image namespace and image naming and tagging convention?